.github/actions/deploy/action.yml

```diff
@@ -7,13 +7,15 @@ inputs:
   aws-secret-access-key:
     description: AWS secret key
     required: true
+  dry-run:
+    description: >-
+      If truthy, the deploy will be a dry run, or will not be attempted at all
+      if `aws-access-key-id` is blank (instead of erroring)
+    required: false
+    default: ""
```

Comment:
question: is this supposed to be a string? I thought there was a boolean type in GitHub Actions inputs.

Reply:
Inputs don't have types — they are always strings.